### DISCUSSION TITLE
Get Cart/Account - GetServerSideProps/GetStaticProps - Next.js

### DISCUSSION THREAD

**Discussion by Lulololu** [swellstores/community#384] [Help]
Hi guys,

I know that cart and account are all retrieved with cookies.
But I would like to know if there is some possibility to retrieve them from GSS or GSP - without custom API.

It would be somehow really convenient to avoid loading state/UI.

Thanks for your help,
Thanks for your great work

**Comment by logeshswell** [comment]
Can you check whether this discussion helps you in solving your usecase?
https://github.com/orgs/swellstores/discussions/122
